template-roles-and-resources.md

```diff
@@ -0,0 +1,167 @@
+# Good Docs Project - Template roles and resources
+
+This document explains the template contributor roles and other tools and resources that support the template creation process. See the [Template Contributing Guide](template-contribuing-guide.md) to understand the larger context for these roles and resources.
+
+## Table of contents
+
+- [Template writing working groups](#template-writing-working-groups)
+- [Templates coordinator](#templates-coordinator)
+- [Template mentors](#template-mentors)
+- [Community reviewers](#community-reviewers)
+- [Pull request reviewers](#pull-request-reviewers)
+
+
+## Template writing working groups
+
+Our project is organized into several working groups that meet on a regular basis to work on the project’s key initiatives.
+One of the best ways to get started with our project is to join and meet with one of our working groups.
+See [The Good Docs Project Working Groups](https://thegooddocsproject.dev/working-group/) for a list of our current active groups.
+
+If you are a first-time contributor, you are required to join a template writing working group as you work on your first template project.
+
+The working groups that are specifically devoted to template writing are:
+
+- [Community Docs](https://thegooddocsproject.dev/working-group/community-docs/)
+- More forthcoming!

```

Comment:
Should we add the APAC/EMEA group here?

Reply:
Yep, that group formed after I submitted this RFC.

Reply:
I adopted this suggestion.